src/Functions/generateSnowflakeID.cpp

```diff
@@ -165,18 +165,29 @@ class FunctionGenerateSnowflakeID : public IFunction
     {
         FunctionArgumentDescriptors mandatory_args;
         FunctionArgumentDescriptors optional_args{
-            {"expr", nullptr, nullptr, "Arbitrary expression"}
+            {"machine_id", nullptr, nullptr, "Optional machine ID"}
         };
         validateFunctionArguments(*this, arguments, mandatory_args, optional_args);
 
         return std::make_shared<DataTypeUInt64>();
     }
 
-    ColumnPtr executeImpl(const ColumnsWithTypeAndName & /*arguments*/, const DataTypePtr &, size_t input_rows_count) const override
+    ColumnPtr executeImpl(const ColumnsWithTypeAndName & arguments, const DataTypePtr &, size_t input_rows_count) const override
     {
         auto col_res = ColumnVector<UInt64>::create();
         typename ColumnVector<UInt64>::Container & vec_to = col_res->getData();
 
+        uint64_t machine_id = 0;
+        if (arguments.size() > 0)
+        {
```

Comment:
Let's remove braces for single-line if-statements.

Reply:
Fixed, please review again, thank you!